### PR TITLE
Fix unused sudo argument for install_gpg_component.sh.

### PR DESCRIPTION
**Pull request by dewyatt**
This was unused previously.

### CODE REVIEW

File: install_gpg_component.sh [hunk 1]
```diff
@@ -285,6 +285,8 @@ if [[ "${_arg_version}" =~ ^latest ]]; then
 	determine_latest_version
 fi
 
+[[ "${_arg_sudo}" = "on" ]] && sudo=sudo || sudo=
```
Comment (skalee) [inline install_gpg_component.sh]: @dewyatt How about moving this condition to `build_and_install()` in order to avoid another global variable?

**Comment by skalee** [comment]
@dewyatt, please note following line:

https://github.com/riboseinc/gpg-build-scripts/blob/35f3979aa25fb5b1d10d8f1f9ccf698c2d712638/install_gpg_component.sh#L296

Even if you install components as ordinary user, you may need to do some final configuration as root.

File: install_gpg_component.sh (hunk 1 above)
Comment (dewyatt) [reply install_gpg_component.sh]: Good idea, done!

**Comment by dewyatt** [comment]
> @dewyatt, please note following line:
> 
> [gpg-build-scripts/install_gpg_component.sh](https://github.com/riboseinc/gpg-build-scripts/blob/35f3979aa25fb5b1d10d8f1f9ccf698c2d712638/install_gpg_component.sh#L296)
> 
> Line 296 in [35f3979](/riboseinc/gpg-build-scripts/commit/35f3979aa25fb5b1d10d8f1f9ccf698c2d712638)
> 
>  sudo tee -a /etc/ld.so.conf.d/gpg2.conf <<<"/usr/local/lib" 
> Even if you install components as ordinary user, you will typically need to do some final configuration as root.

I think this should probably be optional as well. In my use case I may be running as a regular user and just using `LD_LIBRARY_PATH`.

**Comment by skalee** [comment]
Could be, I'm not that proficient in linking in Linux.

**Comment by skalee** [comment]
LGTM. Feel free to merge, unless you want to make that `sudo tee` optional as well. In such case, I'm okay with making execution of this step dependant on `--sudo`/`--no-sudo` option as well.

**Comment by dewyatt** [comment]
Ok I made some slight changes to make the ldconfig and tee lines dependent on the sudo option and updated the help. Maybe worth a re-review.

**Comment by skalee** [comment]
LGTM.

**Pull request merged**